CMakeLists.txt

```diff
@@ -82,7 +82,9 @@ endif()
 
 include(CMakeDependentOption)
 include(CheckIncludeFile)
-include(CTest)
+if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
```

Comment:
I'm surprised the example doesn't make a named variable like `H3_IS_ROOT_PROJECT` - seems like it would help make it clearer what this is testing for

Reply:
Indeed! I'll update the PR.